src/components/SubPage/GenomicCharacterization.tsx

```diff
@@ -1,35 +1,66 @@
 import React from "react";
-import { Flex } from "antd";
 import classNames from "classnames";
-import DiagramCard, { DiagramCardProps } from "../shared/DiagramCard";
+import DiagramCard from "../shared/DiagramCard";
+import { UnpackedGenomicCharacterization as GenomicCharacterizationProps } from "./types";
+import SubpageTable from "../shared/SubpageTable";
+import { DDPCR_COLUMNS, CRRNA_OFF_TARGETS_COLUMNS, AMPLIFIED_JUNCTION_COLUMNS } from "./genomic-characterization-table-constants";
 
 const {
-    card,
-    fullWidth,
+    masonry,
+    masonryItem,
+    masonrySpanAll,
 } = require("../../style/genomic-characterization.module.css");
 
-interface GenomicCharacterizationProps {
-    diagrams: DiagramCardProps[];
-}
-
 const GenomicCharacterization: React.FC<GenomicCharacterizationProps> = ({
     diagrams,
+    amplifiedJunctions,
+    ddpcr,
+    crRnaOffTargets,
 }) => {
+
     return (
-        <Flex gap={40} wrap="wrap" justify="space-between" align="baseline">
+        <div className={masonry}>
+            {amplifiedJunctions?.data.length ? (
+                <SubpageTable
+                    className={masonryItem}
+                    title={"Amplified Junctions"}
+                    columns={AMPLIFIED_JUNCTION_COLUMNS}
+                    dataSource={amplifiedJunctions.data}
+                    caption={amplifiedJunctions.caption}
+                />
+            ) : null}
+            {ddpcr?.data.length ? (
+                <SubpageTable
+                    className={masonryItem}
+                    title={"GFP and donor plasmid copy number"}
+                    columns={DDPCR_COLUMNS}
+                    dataSource={ddpcr.data}
+                    caption={ddpcr.caption}
+                />
+            ) : null}
+            {crRnaOffTargets?.data.length ? (
+                <SubpageTable
```

Comment:
can this subpage table just handle checking for the length of data? so this file doesn't have to have multiple  conditionals. I like that it's either an empty list or a data list, I don't like components having to check if data is of two different types, ie Null or a List, but I think handling an empty list seems ok